vitals/checkBattery.java

```diff
@@ -0,0 +1,43 @@
+package extensions;
+
+public class checkBattery {
+	
+	private static int Min_Temperature = 0;
+	private static int Max_Temperature = 45;
+
+	private static int Min_SOC = 20;
+	private static int Max_SOC = 80;
+
+	private static float Max_Charge_Rate = 0.8f;
+
+	public static boolean batteryIsOk(int temperature, int soc, float chargeRate) 
+	{
+		boolean isTemperature_OK = checkTemperature(temperature);
+		boolean isSOC_OK = checkSOC(soc);
+		boolean isChargeRate_OK = checkChargeRate(chargeRate);
+		

```

Comment:
duplicate println statements and is printing is the responsibility of batteryIsOk()